community/rfcs/24-05-16-OPEA-001-Overall-Design.md

```diff
@@ -38,7 +38,7 @@ This RFC is used to present the OPEA overall design philosophy, including overal
 
 The proposed overall architecture is
 
-![OPEA Architecture](opea_architecture.png "OPEA Architecture")
+![OPEA Architecture](opea_architecture.png)
```

Comment:
What's the difference of adding or not adding this "OPEA Architecture"? It seems the same in the user view.

Reply:
Actually the content in “” works fine without writing, as other locations can attest. But it will interfere with the tool's recognition of the reference. So I removed it.